keyboards/rgbkb/common/common_oled.c

```diff
@@ -10,6 +10,7 @@
 #include "common_oled.h"
 #include "oled_driver.h"
 #include "rgb_matrix.h"
+#include "printf.h"
```

Comment:
```suggestion
#include "print.h"
```

Reply:
doesn't compile:
keyboards/rgbkb/common/common_oled.c: In function 'render_rgb_menu':
keyboards/rgbkb/common/common_oled.c:57:5: error: implicit declaration of function 'snprintf' [-Werror=implicit-function-declaration]

Reply:
Should use `<stdio.h>` then.

Reply:
gotcha, saw the printf.h in the libs dir and thought that was the recommended approach over stdio.h. PR updated.